geometry.zsh

```diff
@@ -270,6 +278,35 @@ prompt_geometry_clear_timestamp() {
   unset prompt_geometry_command_exec_time
 }
 
+GEOMETRY_ASYNC_PROMPT_PROC=0
+prompt_geometry_setup_async_prompt() {
+    function precmd() {
```

Comment:
It doesn't look like it interferes at all. Nevertheless I'm gonna change to use preferred format `add-zsh-hook`.